rfcs.md

```diff
@@ -0,0 +1,35 @@
+# Cauldron RFCs
+
+## What is an RFC?
+
+A request for comments, or further referred to as "RFC" is a process intended to provide a consistent and controlled path for changes to be introduced to cauldron.
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow. Some changes though are "substantial", and we ask that these be put through a bit of a design process and provide a consensus among the cauldron team.
+
+## When to follow this process
+
+You should follow this process if you intend to make "substantial" changes to cauldron or its documentation. Some examples that would benefit from an RFC include:
+
+- A new component or feature
+- Any breaking changes
+
+Some changes that would not need an RFC:
+
+- Bug fixes
+- Refactoring an existing component (while maintaining the same API contract)
+- Rephasing or clarifying documentation
+
+## How to submit an RFC
+
+1. Create a new issue
+1. Add the `rfc` label
+1. Be sure to include descriptive details, this includes:
+   - A descriptive title
+   - Implemention details
+   - Screenshots (if necessary for a new component)
```

Comment:
What do you think about creating an issue template for this?

Reply:
That's a great idea!